app/pages/help.html

```diff
@@ -534,6 +534,8 @@ <h3 id="keeprecentlyclosed">keeprecentlyclosed</h3>
         Toggle whether to keep recently closed tabs and allow them to be restored. This setting is unaffected by other history settings. Disabling this setting will result in the <a href="#action.reopenTab">action.reopenTab</a> being unable to restore previously closed tabs. If <a href="#restoretabs">restoretabs</a> is disabled, all open tabs will be moved to the list of recently closed tabs when quitting Vieb, unless this setting is also disabled (in which case they can not be restored).
         <h3 id="ignorecase">ignorecase</h3>
         Toggle for case-sensitive searching in the page. Any existing search will be updated to match this setting when either <a href="#action.nextSearchMatch">action.nextSearchMatch</a> or <a href="#action.previousSearchMatch">action.previousSearchMatch</a> are called.
+        <h3 id="incsearch">incsearch</h3>
+        Toggle for incremental search in the page.
```

Comment:
Can we include a bit more details about this feature for those not familiar with this Vim setting?

Reply:
Done.